news/623.new.rst

```diff
@@ -0,0 +1 @@
+Add a hook for ``psutil``, which has platform independent exclude list.

```

Comment:
The exclude list is platform-dependent, is it not?

Reply:
ah yes, my bad